run error in jpms and gradle 6.6

My demo project uses gradle 6.6 , jpms structruce and  javafxplugin 0.0.9. When I run then run task, an error occurs as follow
```
> org.joor.ReflectException: java.lang.NoSuchFieldException: javaExecHandleBuilder
```

If I change the gradle version to 6.5 or remove the module-info.java, the run task works fine

> Ok, so I looked into this again because I want to try building JavaFX applications on JDK 16. This will require Gradle 7. I saw that this plugin has a dependency on a plugin to help with the module system (org.javamodularity.moduleplugin). This won't be needed on Gradle 7+ because Gradle 7 will come with full support for Jigsaw modules.
> 
> So I have forked this module on Github to experiment with this, and now I have a working plugin for Gradle 7/JDK 16 that runs my example project perfectly fine.
> 
> If anyone is interested in trying it out, please let me know. I think it might not (yet?) be a good idea to merge back into upstream because it will break things for those using older Gradle versions.

I used your fork in my project but I am getting error 
src\main\java\module-info.java:6: error: module not found: jettison
requires jettison;

Do I need to change some configuration to add this dependancy?

--- a/build.gradle
+++ b/build.gradle
@@ -29,7 +29,7 @@ dependencies {
     implementation gradleApi()
 
     implementation 'com.google.gradle:osdetector-gradle-plugin:1.6.1'
-    implementation 'org.javamodularity:moduleplugin:1.8.0'
+    implementation 'org.javamodularity:moduleplugin:1.8.1'
 
     testImplementation gradleTestKit()
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.2'

--- a/build.gradle
+++ b/build.gradle
@@ -29,7 +29,7 @@ dependencies {
     implementation gradleApi()
 
     implementation 'com.google.gradle:osdetector-gradle-plugin:1.6.1'
-    implementation 'org.javamodularity:moduleplugin:1.7.0'
+    implementation 'org.javamodularity:moduleplugin:1.8.0'
 
     testImplementation gradleTestKit()
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.2'

--- a/build.gradle
+++ b/build.gradle
@@ -29,7 +29,7 @@ dependencies {
     implementation gradleApi()
 
     implementation 'com.google.gradle:osdetector-gradle-plugin:1.6.1'
-    implementation 'org.javamodularity:moduleplugin:1.8.1'
+    implementation 'org.javamodularity:moduleplugin:1.8.2'
 
     testImplementation gradleTestKit()
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.2'

